launch_ros/launch_ros/descriptions/composable_node.py

```diff
@@ -163,6 +165,11 @@ def node_namespace(self) -> Optional[List[Substitution]]:
         """Get node namespace as a sequence of substitutions to be performed."""
         return self.__node_namespace
 
+    @property
+    def node_autostart(self):
+        """Getter for autostart."""
+        return False
+
```

Comment:
It would be great to avoid this addition too.

Reply:
Done :+1: